auth-api/src/auth_api/services/products.py

```diff
@@ -93,14 +93,16 @@ def create_product_subscription(org_id, subscription_data: Dict[str, Any],  # py
                 # create a staff review task for this product subscription if pending status
                 if product_model.default_subscription_status == ProductSubscriptionStatus.PENDING_STAFF_REVIEW.value:
                     user = UserModel.find_by_jwt_token(token=token_info)
-                    task_info = {'name': f'{org.name} - {product_model.description}',
+                    task_type = current_app.config.get('ACCESS_REQUEST_PRODUCT')
```

Comment:
Need some further thinking about how to save these values/descriptions. enum or config or some sort of database table.

Reply:
Refactored them to enums